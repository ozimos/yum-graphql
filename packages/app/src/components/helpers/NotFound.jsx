import React from 'react';
import { Link } from 'react-router-dom';
import '../../../public/styles/dashboard.scss';


import ConnectedNav from '../nav/ConnectedNav';


const NotFound = () => (
  <div id="not-found" className="canvas2">
    <div className="container3">
      <header className="header extra">
        <ConnectedNav />
      </header>
      <div>
        <h3> Sorry, the page you are looking for does not exist</h3>
        <h4> <Link to="/">Go Home</Link></h4>
      </div>
    </div>
  </div>
);

export default NotFound;

