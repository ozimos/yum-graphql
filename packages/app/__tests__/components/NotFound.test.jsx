/* global React:false, shallow:false toJson:false */

import NotFound from '../../src/components/helpers/NotFound';

describe('NotFound Component', () => {

  jest.mock('react-router-dom', () => (
    {
      Link: 'div',
    }
  ));

  it('renders correctly', () => {
    const wrapper = shallow(<NotFound />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});

