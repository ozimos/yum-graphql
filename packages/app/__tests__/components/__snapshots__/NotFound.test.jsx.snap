// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound Component renders correctly 1`] = `
<div
  className="canvas2"
  id="not-found"
>
  <div
    className="container3"
  >
    <header
      className="header extra"
    >
      <Connect(Nav) />
    </header>
    <div>
      <h3>
         Sorry, the page you are looking for does not exist
      </h3>
      <h4>
         
        <Link
          replace={false}
          to="/"
        >
          Go Home
        </Link>
      </h4>
    </div>
  </div>
</div>
`;
